core/relay/relay-docker.md

```diff
@@ -169,14 +169,14 @@ First **login** to github container registry:
 
 * Copy `.env.template.2` file to `.env`.
 * Keep following Environment Variables in `.env` and delete/ignore rest:
-    * RELAY_NAME: Name for the relay.
+    * RELAY_NAME: Name for the relay. Same is used for container name.
```

Comment:
As above.

Reply:
Hi Rama, this set of instructions doesn't involve any config file. (It gets generated). So not sure if same above comment would apply here?
Here we are explicitly setting the Relay Name for the first time.